src/components/ImageView/Overlay/OverlayComponent.tsx

```diff
@@ -72,10 +72,6 @@ export class OverlayComponent extends React.Component<OverlayComponentProps> {
 
     render() {
         const styleString = this.props.overlaySettings.styleString;
-        const frameView = this.props.frame.requiredFrameView;
```

Comment:
I think those are in there to ensure that changing the frame view, padding or width/height triggers a re-render

Reply:
oh! sorry about that, I'll restore those codes